xarray/backends/plugins.py

```diff
@@ -72,7 +73,9 @@ def backends_dict_from_pkg(
             backend = entrypoint.load()
             backend_entrypoints[name] = backend
         except Exception as ex:
-            warnings.warn(f"Engine {name!r} loading failed:\n{ex}", RuntimeWarning)
+            warnings.warn(
```

Comment:
It would be better to replace all of these with `utils.emit_user_level_warning`

Reply:
(I also think it's fine to do this in another PR, though defer to @dcherian)

We could also add a regex lint in pre-commit that we don't use `warnings.warn`, we only use `utils.emit_level_warning`. Here's an example of this from another repo: https://github.com/PRQL/prql/blob/main/.pre-commit-config.yaml#L64-L71

Reply:
> I also think it's fine to do this in another PR,

yes fine by me!